source/stylesheets/_tab-secrets-env_vars.scss.erb

```diff
@@ -84,21 +86,127 @@
 							&.ng-invalid:not(.ng-pristine) {
 								@include input_key_styling(true);
 							}
+
+							&:disabled {
+								opacity: 0.5;
+							}
 						}
 
-						textarea.value {
+						.env-var-value {
 							@include order(3);
 							@include flex-grow(1);
-							@include align-self(stretch);
-							@include box-sizing(border-box);
-							border: 1px solid $def-mediumlightgrey;
-							@include border-radius($def-radius);
-							@include padding-y(8px);
+							@include align-self(flex-start);
+							position: relative;
+
+							textarea.value {
+								@include align-self(stretch);
+								@include box-sizing(border-box);
+								border: 1px solid $def-mediumlightgrey;
+								@include border-radius($def-radius);
```

Comment:
Move border and radius to the container, along with overflow: hidden - this will be a neater solution when it comes to the eye icon - see below.